eds/blocks/partners-navigation/utilities/utilities.js

```diff
@@ -1,8 +1,12 @@
-// MWPW-159021
+// PARTNERS_NAVIGATION START
+// MWPW-159021 - Fix eslint errors
 /* eslint-disable */
-// MWPW-157751
-import { getLibs } from '../../../scripts/utils.js';
+// PARTNERS_NAVIGATION END
+/* eslint import/no-relative-packages: 0 */
 
+// PARTNERS_NAVIGATION START
+// MWPW-157751 - Text is visible through Gnav when scrolling on mobile view
```

Comment:
I don't think this is quite right, the getLibs must be original milo code. I think
// PARTNERS_NAVIGATION START
should only start at L25, wdyt?

Reply:
current markers are ok